posts/2022-04-12-java18-22004.adoc

```diff
@@ -25,6 +25,7 @@ In link:{url-about}[Open Liberty] 22.0.0.4:
 * <<certificate, Support client certificate with URL encoded strings>>
 * <<CVEs, Security Vulnerability (CVE) Fixes>>
 * <<bugs, Notable bug fixes>>
+* <<guides, New and updated guides since the previous release>>
 * <<paketo, Introducing the Paketo Liberty Buildpack>>
```

Comment:
```suggestion
* <<paketo, Introducing the Paketo Liberty Buildpack>>
* <<CVEs, Security Vulnerability (CVE) Fixes>>
* <<bugs, Notable bug fixes>>
```

Reply:
We don't need the link to the guides section, as it's called out in the sentence below the ToC (and guides are not part of the runtime)